test/smart_city/organization_user_associate_test.exs

```diff
@@ -0,0 +1,54 @@
+defmodule SmartCity.OrganizationUserAssociateTest do
+  use ExUnit.Case
+  use Placebo
+  alias SmartCity.OrganizationUserAssociate
+
+  setup do
+    string_key_message = %{
+      "user_id" => "Nathaniel",
+      "org_id" => 1
+    }
+
+    json_message = Jason.encode!(string_key_message)
+
+    atom_key_message = %{user_id: "Nathaniel", org_id: 1}
+
+    invalid_message = %{user_id: "Nathaniel"}
+
+    {:ok,
+     string_key_message: string_key_message,
+     json_message: json_message,
+     atom_key_message: atom_key_message,
+     invalid_message: invalid_message}
```

Comment:
If all these guys are each only used once I'd prefer they only exist in their respective tests.